### DISCUSSION TITLE
Why hasn't it been updated for a long time?

### DISCUSSION THREAD

**Discussion by misaka10843** [kancolle-ohuraiN-project/kancolle-ohuraiN#2] [Announcements|公告]
If you pay attention, you'll notice that I haven't updated in a long time.
The reason is simple: I'm not a big fan of kancolle, and my Unity level isn't very good, and I have my own studies, so I can't do full time development.
I've thought about finding some like-minded partners to develop with, but I can't find anyone (or I can't promote myself).

Another reason is that I have to develop houbun-fantasia, a 2D RPG (although there is no art production), but I still need to focus on the same game as much as possible (and also find time to learn to strengthen the painting and music)

That's why there is less and less energy to update, please understand

---

如果注意的话，你会发现我已经很久没有更新了。
其实原因很简单，我并不是kancolle的忠实粉丝，并且我的Unity水平并不是很好，而且我还有自己的学业，所以无法做到全职开发。
虽然也想过找些志同道合的伙伴一起开发，但是根本找不到人（还是自己不会宣传）

还有一个原因是因为我还要开发houbun-fantasia这款2D RPG游戏（虽然现在连美术制作都没有），但是还是需要将精力尽量放在同人游戏中（并且还会抽时间学习强化绘画与音乐）

所以才越来越没什么精力更新，还请谅解

**Comment by cvakagi** [comment]
大佬我支持你，我半路进坑2.0退坑。前面错过了好多活动，后面也没咋玩，很想再一次体验打活动那种感觉

**Comment by PoiLiang** [comment]
您好，請問我可以加入這個項目的開發嗎?
目前自行製作的部分是主畫面的UI互動以及資源顯示，如果可以的話想為這個項目繼續更新

**Reply by misaka10843** [reply]
您好，对不起这么晚了才回复，因为现在colle的话还在运行中（并且没有停服的迹象），所以项目还在停止之中（防止被警告）
十分感谢您有意愿参与此项目，如果项目重新开启时我会重新联系您，十分感谢qwq
